src/meta/src/hummock/manager/mod.rs

```diff
@@ -2070,7 +2064,7 @@ fn drop_sst(
                     false
                 }
             } else {
-                false
+                true
```

Comment:
Shall we panic here?

Reply:
Agree

Reply:
No, it may be change by another split operation.

Reply:
Oh, I made a mistake, we have checked it before.